File: Bridge Python Client/metamenth/enumerations/meter_type.py
from enum import Enum


class MeterType(Enum):
    """
    Different types of meters used in a building.

    Author: Peter Yefi
    Email: peteryefi@gmail.com
    """
    ELECTRICITY = "Electricity"
    CHARGE_DISCHARGE = "ChargeDischarge"
    POWER = "Power"
    FLOW = "Flow"
    HEAT = "Heat"
    GAS = "Gas"

File: Bridge Java Server/src/main/java/com/middleware/metamenth/enums/MeterType.java
package com.middleware.metamenth.enums;

public enum MeterType {
    ELECTRICITY("Electricity"),
    CHARGE_DISCHARGE("ChargeDischarge"),
    POWER("Power"),
    FLOW("Flow"),
    HEAT("Heat"),
    GAS("Gas");

    private final String value;

    MeterType(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}
